c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -159,10 +159,18 @@ vars:
     # this application is a child application in a parent/child
     # architecture.
     external_themes_url:
-    external_mapserv_url:
-    # If the WFS service is different from the WMS service, uncomment and set the
-    # external_mapserv_wfs_url parameter
-    #external_mapserv_wfs_url:
+
+    mapserverproxy:
```

Comment:
For me it's included in this point
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl#L7-L24
but I just see thet the `vars_project.yaml` is missing ...